docker-compose.yml

```diff
@@ -4,7 +4,7 @@ services:
   localstack:
     image: localstack/localstack:latest
     environment:
-      - SERVICES=s3
+      - SERVICES=s3,sqs
```

Comment:
are we using localstack sqs in the end?

Reply:
ahh, no we're not, good spot, thanks, I've removed that in https://github.com/guardian/newswires/pull/83/commits/1df356325c7249eeb2613e647f2b6f4c823bc394